### ISSUE TITLE
SRB plume very big for small diameter SRBs

### ISSUE THREAD

**Issue by jrodrigv** [Tidal-Stream/ProceduralParts#3]
Hi @Tidal-Stream , first of all thank you for taking over the maintenance of this great mod.

I would like to request a smaller particle model for small diameter SRB ( between 0.1m - 0.2m of diameter)

I use this mod as dependency for BDArmory Modular missiles. And usually missiles have a diameter smaller than more typical  SRBs

You can see on this video exactly what I'm talking about:

https://www.youtube.com/watch?v=KzN78oBWsho

**Comment by Tidal-Stream** [comment]
HI @jrodrigv. By particle model do you mean the plume? I am not sure how to do this but if someone figures it out, I will merge the patch.

**Comment by jrodrigv** [comment]
Yes, I'm not sure how it works for ProceduralParts but I guess that depending on the thrust the exhaust of the SRB should be scaled.

If the SRB plume is already dynamic depending on the Thrust then a new model is not required and I guess that it just a matter of reviewing the parameters on the code used for scaling up/down the plume/exhaust.
